Source/JavaScriptCore/wasm/WasmAirIRGenerator.cpp

```diff
@@ -555,70 +647,69 @@ class AirIRGenerator {
 
     B3::PatchpointValue* addPatchpoint(B3::Type type)
     {
+#if USE(JSVALUE32_64) && ASSERT_ENABLED
+        for (B3::Type kind : (type.isTuple() ? m_proc.tupleForType(type) : Vector<B3::Type> { type }))
+            ASSERT(kind != B3::Int64);
```

Comment:
How can we assert this?

Reply:
I actually do think it's a property of this patch that on 32-bit platforms air wants all the integers to be Int32; we should probably talk more about if this is a property we want to have or not and if it should be made more explicit.